scripts/write_deployinst_input.py

```diff
@@ -336,7 +390,7 @@ def region_render(list, template, full_template, output_file):
         number = ''
         lifetime = ''
 
-        for data in list:
+        for data in data_list:
```

Comment:
why are we looping over data list again?
still meaningless name. names should have information in them.